python/friesian/example/recall/README.md

```diff
@@ -1,6 +1,100 @@
-https://github.com/facebookresearch/faiss/blob/main/INSTALL.md
+# Recall items by using faiss
+This example demonstrates how to use BigDL Friesian 
+to get the recalled items by using the retrieval algorithms provided by faiss.
```

Comment:
modify this to add the keywords: offline recall, efficiently, spark

Reply:
got it